backend/graphql_v1/universal_codes/src/types/entity_search_input.rs

```diff
@@ -31,14 +31,15 @@ impl From<EntitySearchInput> for EntitySearchFilter {
     fn from(f: EntitySearchInput) -> Self {
         EntitySearchFilter {
             code: f.code,
-            description: f.description,
+            // many descriptions contain slashes, we need to escape them to provide a valid search input
+            description: f.description.map(|d| d.replace("/", "\\/")),
```

Comment:
I wonder what other characters we might need to escape...
This is great for now though!